packages/patterns/compiler.tsx

```diff
@@ -58,11 +58,11 @@ export default recipe<Input>(
       [NAME]: "My First Compiler",
       [UI]: (
         <div>
-          <common-code-editor
-            source={code}
+          <ct-code-editor
+            value={code}
             language="text/x.typescript"
             onChange={updateCode({ code })}
-            errors={errors}
+            //errors={errors}
```

Comment:
I ran into this too, I feel like it would be better to retain the error support but... I didn't write those codemirror extensions and updating them feels like a rabbithole.